projects/gov-ui/src/lib/components/gov-uk-checkbox/gov-uk-checkbox.component.ts

```diff
@@ -21,7 +21,8 @@ import {FormGroup} from '@angular/forms';
 export class GovUkCheckboxComponent implements OnInit {
   constructor() { }
   @Input() group: FormGroup;
-  @Input() config: {value: string, label: string, hint: string; name: string; focusOn: string; id: string; classes: string};
+  @Input() config: {value: string, label: string, hint: string; name: string; 
+                    focusOn: string; id: string; classes: string; hiddenLabelContext: string};
```

Comment:
This would be much more readable:
```
@Input() config: {
  value: string;
  label: string;
  hint: string;
  name: string;
  focusOn: string;
  id: string;
  classes: string;
  hiddenLabelContext: string
};
```
And now that I write it out like that, I realised I missed out that there was a mixture of commas and semicolons in that declaration! To be fair, either _all_ commas or _all_ semicolons is acceptable. I think the more "typescript" way of doing it is with semicolons... but I don't know which is more consistent with the rest of the application.

In _theory_, this should be declared as an interface somewhere rather than being inline but it's not the end of the world... just so long as they're all the same separators. :)

Reply:
formatted